plugins/janus_audiobridge.c

```diff
@@ -1919,8 +1924,25 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->ssrc = 0;
 		pkt->timestamp = ntohl(rtp->timestamp);
 		pkt->seq_number = ntohs(rtp->seq_number);
+		/* Any audio level extension to quickly check if this is silence? */
+		pkt->silence = FALSE;
+		if(participant->extmap_id > 0) {
+			int level = 0;
+			if(janus_rtp_header_extension_parse_audio_level(buf, len, participant->extmap_id, &level) == 0) {
+				pkt->silence = (level == 127);
```

Comment:
For time being we use some threshold to decide silence
pkt->silence = (level > configurable_threshold)?TRUE:FALSE;

the famous [hark](https://github.com/otalk/hark/blob/master/hark.bundle.js#L58) script is using -50dBov as threshold
-127 to -50 is silence, -50 to -0 is voice, we can put 100dBov as less impacted threshold.

Here we need to maintain some history of previous volume levels, then we can make more accurate decisions.